### PR TITLE
ISPN-10069 docs for the operator tutorial

### PR DESCRIPTION
**Pull request by oraNod**

### CODE REVIEW

File: operator/README.md [hunk 1]
```diff
@@ -1,67 +1,65 @@
...
-----------------
-You should have a running installation of an OpenShift version that supports running Operators.
-At the time of writing, OpenShift 4 or 3.11 support running them.
+Installing the Infinispan Operator
+----------------------------------
 
-To go through this tutorial, you must have admin access to the OpenShift cluster.
-This is because installing Operators require admin privileges.
+* Install the Infinispan Operator from [OperatorHub.io](https://operatorhub.io/).
+* Manually install the Infinispan Operator. See the [Infinispan Operator README](https://github.com/infinispan/infinispan-operator).
```
[the remark this answers is not in the document]
Comment (oraNod) [reply operator/README.md]: @galderz good point, I've revised. hopefully clearer now.

[the remark this answers is not in the document]
Comment (galderz) [reply operator/README.md]: I don't think it's clearer, but this time the problem is that we're losing information. 

The https://github.com/infinispan/infinispan-operator repo README does not explain how to install the operator manually in the same way that the existing tutorial README does.

[the remark this answers is not in the document]
Comment (oraNod) [reply operator/README.md]: Trying again, @galderz I've added the steps to install manually and used `${tag}` in the URLs so users don't just copy/paste and end up with an older version.

**Comment by karesti** [comment]
@oraNod need rebase

**Comment by karesti** [comment]
@galderz anything else to do here ?

**Comment by galderz** [comment]
@karesti @oraNod Looks good now, integrating.

**Comment by galderz** [comment]
Thx @oraNod :)

**Pull request merged**